docs/tutorials/bids_archive_tutorial.ipynb

```diff
@@ -0,0 +1,755 @@
+    "2. **Full Name**: Up to several words, fully describes the entity. (e.g., 'Contrast Enhancing Agent', 'Subject')\n",
+    "3. **File Name Key**: A few characters, used in file names. (e.g., 'ce', 'sub')\n",
+    "\n",
+    "A few entities and their multiple representations are shown in the table below:\n",
+    "\n",
+    "| Entity | Full Name | File Name Key |\n",
+    "| --- | --- | --- |\n",
+    "| ceagent | Contrast Enhancing Agent | ce |\n",
+    "| subject | Subject | sub |\n",
+    "| session | Session | ses |\n",
+    "| run | Run | run |\n",
+    "\n",
+    "As of this writing, the other valid entities for BIDS and BIDS Derivatives are listed in the BIDS Standard and in JSON files in the PyBids Github repository (https://github.com/bids-standard/pybids/tree/master/bids/layout/config). An easy way to view all valid entities in RT-Cloud, which are loaded from PyBids, is to view the dictionary returned by the following:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "entities = loadBidsEntities()\n",
+    "print(\"Entities:\", entities.keys())"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "There are also a few entities with only one representation, which aren't used in a key-value form. Examples include `datatype` (e.g., 'func' or 'anat'), `extension` (e.g., '.nii', '.nii.gz', '.json'), and `suffix` (e.g., 'bold'). See the table below for how these can appear in file naming and archive organization.\n",
+    "\n",
+    "Together, these entities provide a unique and consistent way to name files and organize the BIDS dataset.\n",
+    "\n",
+    "#### Exercise: What entities are present in the path `sub-01/func/sub-01_task-languageproduction_run-01_bold.nii.gz`, and what are the entity values? \n",
+    "\n",
+    "##### Answer: \n",
+    "| Entity Name | Value |\n",
+    "|---          | ---   |\n",
+    "| subject | 01 |\n",
+    "| datatype | func |\n",
+    "| task | languageproduction |\n",
+    "| run | 01 |\n",
+    "| suffix | bold |\n",
+    "| extension | .nii.gz|"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": []
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# BIDS Archive: Opening Existing Dataset\n",
+    "\n",
+    "Objective: Learn how to create a BIDS Archive pointing to a specific dataset on disk.\n",
+    "\n",
+    "Procedure:\n",
+    "1. Download a small, sample dataset from OpenNeuro to use with `BidsArchive`.\n",
+    "2. Open the dataset using `BidsArchive` and print out some summary data about it"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "scrolled": false
+   },
+   "outputs": [],
+   "source": [
+    "# https://openneuro.org/datasets/ds002014/versions/1.0.1/download -- <40MB dataset\n",
+    "shutil.rmtree(TARGET_DIR)\n",
```

Comment:
I needed to add ignore_errors=True because the directory didn't exist at first
shutil.rmtree(TARGET_DIR, ignore_errors=True)